### PR TITLE
Add and use Storage, Holdable, Serializable.

### PR DESCRIPTION
**Pull request by wcarss**
Now the player will catch an object that's thrown at them.

### CODE REVIEW

File: Assets/Scripts/Player/Interaction/PickupItem.cs [hunk 1]
```diff
@@ -11,60 +11,89 @@ public class PickupItem : NetworkBehaviour
...
             lookingAt.Add(other);
+        }
     }
 
     private void OnTriggerExit(Collider other)
     {
         if (other.GetComponent<Pickupable>())
+        {
             lookingAt.Remove(other);
+        }
```
Comment (murrelljenna) [inline Assets/Scripts/Player/Interaction/PickupItem.cs:96]: But me me like if statements with no curly braces

**Comment by murrelljenna** [comment]
So there's no separate HandStorage that extends Storage anymore?

**Review by murrelljenna — APPROVED** [review]
Me me like

File: Assets/Scripts/Player/Interaction/PickupItem.cs (hunk 1 above)
Comment (wcarss) [reply Assets/Scripts/Player/Interaction/PickupItem.cs:96]: those are danger 💀

**Pull request merged**